src/config/knexfile.js

```diff
@@ -15,7 +15,8 @@ module.exports = {
     },
     migrations: {
       directory: '../data/migrations'
-    }
+    },
+    debug: true
```

Comment:
```suggestion
    debug: process.env.NODE_ENV === "development"
```
Esse setup é usado somente por dev ou também para prod?

Reply:
De momento, para dev
Mas pra não correr o risco de esquecermos desse detalhe lá na frente, farei a mudança